### DISCUSSION TITLE
buddhism/2023/noble_truths/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/buddhism#76] [General]
# buddhism/2023/noble_truths/

https://shanenull.com/buddhism/2023/noble_truths/

**Comment by shane0** [comment]
https://shanenull.com/buddhism/2023/noble_truths/
https://shanenull.com/buddhism/2024/dukkha/
https://shanenull.com/buddhism/2023/origination/

**Comment by shane0** [comment]
https://github.com/shane0/buddhism/discussions/39
